bin/npg_delete_dr_runs

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Carp;
+use English qw(-no_match_vars);
+use Getopt::Long;
+use Pod::Usage;
+
+use npg_tracking::Schema;
+
+our $VERSION = '0';
+
+my $help;
+my $files;
+my $folders;
+my $no_dry_run;
+
+sub send_commands {
+    my ($dr_server, $cmd) = @_;
+    open(BASH, "| ssh $dr_server bash -e") or die("Can't run ssh: $!");
```

Comment:
Would prefer the `open FILEHANDLE,MODE,EXPR,LIST` form
We use `croak` elsewhere in this script.
```suggestion
    open(BASH, '|-', 'ssh', $dr_server, qw(bash -e)) or croak("Can't run ssh: $!");
```
(untested)